valispace/__init__.py

```diff
@@ -62,18 +62,31 @@ def __init__(self, url=None, username=None, password=None):
 		except:
 			print "VALISPACE-ERROR: Invalid credentials or url"
```

Comment:
Change all print statements to print functions to be compatible with Python 3: print "x" -> print("x")